<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiplayerModeWindow</class>
 <widget class="QDialog" name="MultiplayerModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выберите режим</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #0a2463, stop:1 #3e92cc);
    border: 3px solid #1e3d6b;
}
QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1e3d6b, stop:1 #0a2463);
    border: 2px solid #d4af37; border-radius: 15px; padding: 12px 24px;
    font-family: &quot;Arial Rounded MT Bold&quot;; font-size: 18px; color: #f8f8ff;
    min-width: 200px; min-height: 50px;
}
QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #3a5f8c, stop:1 #1e3d6b);
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnHost">
     <property name="text">
      <string>Создать игру (Хост)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnJoin">
     <property name="text">
      <string>Присоединиться к игре</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBack">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
